src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { Block } from 'baseui/block';
+import { Button, SIZE } from 'baseui/button';
+import { Checkbox, LABEL_PLACEMENT } from 'baseui/checkbox';
+import { FormControl } from 'baseui/form-control';
+import { Input } from 'baseui/input';
+import { HeadingMedium } from 'baseui/typography';
+import React from 'react';
+
+import { useLoginForm } from '../../../hooks';
+
+export default function LoginForm(): React.ReactElement {
+  const { formik, success, error } = useLoginForm();
+
+  return (
+      <form onSubmit={formik.handleSubmit}>
+        {success ? <h2 id="success">SUCCESS!</h2> : null}
+        {error ? <h2 id="error">ERROR!</h2> : null}
+        <HeadingMedium
+          color="fontColor"
+          marginBottom="scale600"
+        >
+          Please sign in
+        </HeadingMedium>
+
+        <Block>
+          <FormControl
+            label={() => 'Email address'}
+            error={formik.errors.username}
+          >
+            <Input
+              overrides={{
+                Root: {
+                  props: {
+                    'data-testid': 'username',
+                  },
+                },
+                Input: {
+                  style: ({ $theme }) => ({
+                    backgroundColor: `${$theme.colors.inputBackground}`,
+                    color: `${$theme.colors.fontColor}`,
+                    caretColor: `${$theme.colors.fontColor}`,
+                  }),
+                },
+              }}
+              name="username"
+              placeholder="Email address"
+              value={formik.values.username}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              autoComplete="off"
+            />
+          </FormControl>
+        </Block>
+        <Block>
+          <FormControl label={() => 'Password'} error={formik.errors.password}>
+            <Input
+              overrides={{
+                Root: {
+                  props: {
+                    'data-testid': 'password',
+                  },
+                },
+                Input: {
+                  style: ({ $theme }) => ({
+                    backgroundColor: `${$theme.colors.inputBackground}`,
+                    color: `${$theme.colors.fontColor}`,
+                    caretColor: `${$theme.colors.fontColor}`,
+                  }),
+                },
+              }}
+              name="password"
+              placeholder="Password"
+              value={formik.values.password}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              autoComplete="off"
+              type="password"
+            />
+          </FormControl>
+        </Block>
+        <Block style={{ marginTop: '1rem', marginBottom: '1rem' }}>
```

Comment:
Use theme spacing class to control margins and paddings.

Also I believe you want to set rowGap on the Block that contains the input form to create consistent spacing between each bloc element.

Reply:
tried using `GridRowGap` but it didn't worked, so used theme spacing to make it work.